pkg/storaged/filesystem/mismounting.jsx

```diff
@@ -105,6 +137,9 @@ export const MismountAlert = ({ warning, fstab_config, forced_options, backing_b
             opts.push("nofail");
         if (opt_netdev)
             opts.push("_netdev");
+        if (subvol) {
+            opts.push(`subvol=${subvol.pathname}`);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20231205-210454-49836fdb-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)